helm/prometheus-rules/templates/alerting-rules/flux.rules.yml

```
@@ -114,3 +114,15 @@ spec:
         severity: page
         team: honeybadger
         topic: releng
+    - alert: GSFluxSuspendedForTooLong
+      annotations:
+        description: |-
+          {{`Flux {{ $labels.kind }} {{ $labels.name }} in ns giantswarm on {{ $labels.installation }} has been suspended for 24h.`}}
+      expr: gotk_suspend_status{namespace="flux-giantswarm"} > 0
+      for: 24h
+      labels:
+        area: kaas
+        cancel_if_outside_working_hours: "true"
+        severity: notify
```

```suggestion
        severity: page
```

I think we should page but just during business hours. Otherwise we'll miss this.

---

got it, I'll change it